packages/plan-form/src/Form/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,5 +1,162 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`containers/forms/PlanForm Form submission for editing plans works: invalid data type Errors 1`] = `
+Array [
+  Object {
+    "errors": Array [
+      "'interventionType' must be one of [Dynamic-FI, Dynamic-IRS, Dynamic-MDA, FI, IRS, IRS-Lite, MDA, MDA-Point, SM]",
+    ],
+    "name": Array [
```

Comment:
This is wierd, would you happen to know what happened here?

Reply:
fixed